homeassistant/components/awair/sensor.py

```diff
@@ -74,6 +76,16 @@
         "unit_of_measurement": "µg/m3",
         "icon": "mdi:cloud",
     },
+    "LUX": {
+        "device_class": DEVICE_CLASS_ILLUMINANCE,
+        "unit_of_measurement": "lx",
+        "icon": "mdi:weather-sunny",
+    },
+    "SPL_A": {
+        "device_class": DEVICE_CLASS_DECIBELS,
```

Comment:
This device class doesn't exist in the base sensor entity component. This platform seems to be using the device class property in a non standard way. We shouldn't extend this usage but rather fix it. It's ok to use it for device type but we shouldn't return it as device_class unless it's part of the sensor device classes.

https://github.com/home-assistant/home-assistant/blob/9df2c3f8c9c34f1014dbb4c69362588e963eb9d5/homeassistant/components/sensor/__init__.py#L34-L43

Reply:
Yes its actually measuring sound level but since its reported in db it could be appropriate to use `DEVICE_CLASS_SIGNAL_STRENGTH`?

Reply:
If it's a sound level and not signal strength we can't use the signal strength device class.

Reply:
Sure I will create a separate PR to support sound level

Reply:
It's ok to return `None` for device_class for the sensor types that doesn't have a corresponding device class yet.